### PR TITLE
SHIM3-298 update scripts to Python 3.10.11 and tidy up

### PR DESCRIPTION
**Pull request by marknolan**

### CODE REVIEW

File: LogAndStream/python_scripts/Bluetooth commands/aAccel5Hz.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import sys
+
+
+def wait_for_ack():
+    ddata = ""
+    ack = struct.pack('B', 0xff)
+    while ddata != ack:
+        ddata = ser.read(1)
+        print("0x%02x" % ord(ddata[0]))
+
```
Comment (rameshchh) [inline LogAndStream/python_scripts/Bluetooth commands/aAccel5Hz.py:13]: extra line

File: LogAndStream/python_scripts/Bluetooth commands/getStatus.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+import time
+
+
+def wait_for_ack():
+    ddata = ""
+    ack = struct.pack('B', 0xff)
+    while ddata != ack:
+        ddata = ser.read(1)
+        print("0x%02x" % ord(ddata[0]))
+
```
Comment (rameshchh) [inline LogAndStream/python_scripts/Bluetooth commands/getStatus.py:15]: extra line

File: LogAndStream/python_scripts/Bluetooth commands/s1_aAccel5Hz-sdbt.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python
+import struct
+import sys
+
```
Comment (rameshchh) [inline LogAndStream/python_scripts/Bluetooth commands/s1_aAccel5Hz-sdbt.py:4]: extra line.

File: LogAndStream/python_scripts/Bluetooth commands/s1_aAccel5Hz-sdbt.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+import serial
+
+
+def wait_for_ack():
+    ddata = ""
+    ack = struct.pack('B', 0xff)
+    while ddata != ack:
+        ddata = ser.read(1)
+        print("0x%02x" % ord(ddata[0]))
+
```
Comment (rameshchh) [inline LogAndStream/python_scripts/Bluetooth commands/s1_aAccel5Hz-sdbt.py:14]: extra line.

File: LogAndStream/python_scripts/Bluetooth commands/s1_aAccel5Hz-sdbt.py (hunk 3 above)
Comment (marknolan) [reply LogAndStream/python_scripts/Bluetooth commands/s1_aAccel5Hz-sdbt.py:4]: Pycharm sorts the imports like this when the file is 'tidied'

File: LogAndStream/python_scripts/Bluetooth commands/s1_aAccel5Hz-sdbt.py (hunk 4 above)
Comment (marknolan) [reply LogAndStream/python_scripts/Bluetooth commands/s1_aAccel5Hz-sdbt.py]: done

File: LogAndStream/python_scripts/Bluetooth commands/getStatus.py (hunk 2 above)
Comment (marknolan) [reply LogAndStream/python_scripts/Bluetooth commands/getStatus.py]: done

File: LogAndStream/python_scripts/Bluetooth commands/aAccel5Hz.py (hunk 1 above)
Comment (marknolan) [reply LogAndStream/python_scripts/Bluetooth commands/aAccel5Hz.py]: done

**Review by rameshchh — APPROVED** [review]

**Pull request merged**